### ISSUE TITLE
Program requirements

### ISSUE THREAD

**Issue by vatterspun** [alexporrello/TWBoilerplateMacros#3]
In testing I had some difficulty getting the program to run.  Do you need a specific document template or acronym table?

Setup: Win10x64, Word 2016

Steps:

1. Download batch file and run
2. Test on a document with some acronyms

Defect:

No change

**Comment by alexporrello** [comment]
To answer your question, yes, you must have a two-column acronym table. The top left "cell" of the table must say "Acronym," "Acronyms," "Abbreviation," or "Abbreviations." Can you please advise on what type of acronym table you're using? This macro will run regardless of your document's template.

Also, in your steps you said nothing about running the macro. Did you run it? It will not automatically run when you open Word.

**Comment by vatterspun** [comment]
> Did you run it?

Yeah I should have been more explicit.  I'll try to update with more details this weekend.
